sdk/identity/identity/src/credentials/clientSecretCredential.ts

```diff
@@ -41,7 +42,7 @@ export class ClientSecretCredential implements TokenCredential {
     clientSecret: string,
     options?: TokenCredentialOptions
   ) {
-    this.identityClient = new IdentityClient(options);
+    this.identityClient = new IdentityClient({ ...options });
```

Comment:
I don't see the spread operator being used on the options when instantiating the IdentityClient in other credential implementations like `ClientCertificateCredentail` or `AuthorizationCodeCredential`. What is the reason for doing this only for `ClientSecretCredential`?

Reply:
Just an oversight! I’ll clean this up

Reply:
Same for ManagedIdentityCredential